### DISCUSSION TITLE
Welcome to sqlite-jdbc Discussions!

### DISCUSSION THREAD

**Discussion by xerial** [xerial/sqlite-jdbc#572] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by itachi-19** [comment]
I can't find prebuilt jar files anywhere and I don't know much about building from sources. Could someone please help me out in making a jar file of the jdbc driver so that I could import in my projects? Thanks in advance 😃

**Reply by gotson** [reply]
https://github.com/xerial/sqlite-jdbc/releases or https://repo1.maven.org/maven2/org/xerial/sqlite-jdbc/

**Comment by ThomasGustafson** [comment]
Is there anywhere I can download a sqlite jdbc jar that is sqlite version 3.6.19 or newer so that my program can use foreign key constraints?

Thank you for your time and any help 

Thomas Gustafson

**Reply by xerial** [reply]
https://github.com/xerial/sqlite-jdbc/releases or https://repo1.maven.org/maven2/org/xerial/sqlite-jdbc/

**Comment by DejaVuMan** [comment]
Out of curiosity, what Java versions does this support? I'm still new to the Java ecosystem, so I have no clue if this will work with Java 11 or if Java 8 is absolutely required. Java 11 should be backwards compatible, so there should be no problems, right?

**Reply by gotson** [reply]
This requires Java 8 as a minimum

**Reply by sinb4d** [reply]
Hi all, the new versions of the driver is failing with "Connection failed: org/slf4j/LoggerFactory (jdbc:sqlite:<File> using org.sqlite.JDBC)" first time after adding it to Netbeans 22, Java JDK 22, in Services as a new driver, everytime after a second attempt to connect to a database it runs into a different exception : "Connection failed: Could not initialize class org.sqlite.JDBC (jdbc:sqlite:<FILE> using org.sqlite.JDBC) "

I have tried older versions with no succes. Older versions can show different errors when the path of the database is correct or wrong, but the latest versions are not giving enough information for debugging. There is absolutely no way of knowing what could have gone wrong.

on the other hand, the connection works in code.

Thanks in advance for your help and time for reading this.
